.ci/watchdog/ci_watchdog.groovy

```diff
@@ -0,0 +1,44 @@
+node(LABEL) {
+    try {
+            BUILD_WORKSPACE="$WORKSPACE/$BUILD_NUMBER"
+            stage("Clone repository") {
+                dir ("$BUILD_WORKSPACE") {
+                    checkout([$class: 'GitSCM', branches: [[name: "*/$BRANCH"]], 
+                            doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', timeout: 30]],  submoduleCfg: [],
+                            userRemoteConfigs: [[credentialsId: '6887a177-8c4d-4fe9-9c3b-fcd71b22bfba', url: 'git@github.com:NervanaSystems/ngraph-onnx.git']]])
+                }
+            }
+            stage("Prepare environment") {
+                dir ("$BUILD_WORKSPACE") {
+                    sh """#!/bin/bash
+                        virtualenv -p python2 venv
```

Comment:
Let's avoid using Python2. End of life for Python 2 is in a little over a year, we're hoping this code will live longer.
https://pythonclock.org/